src/pfe/file-watcher/server/src/controllers/projectStatusController.ts

```diff
@@ -530,10 +532,13 @@ function pingInTransitApplications(): void {
                                         data.detailedAppStatus = detailedAppStatus;
                                         pingCountMap.delete(projectID);
                                     } else if (newState === AppState.starting && pingCount == pingCountLimit) {
+                                        const troubleShootLinkLabel = await locale.getTranslation("projectStatusController.linkLabel");
                                         detailedAppStatus = {
                                             severity: "WARN",
                                             message: newMsg,
-                                            notify: true
+                                            notify: true,
+                                            linkLabel: troubleShootLinkLabel,
```

Comment:
Do we need updates to the unit/functional tests to validate that these events have the `linkLabel` and `link` properties when `notify` is `true`?

Reply:
We can handle this in the subsequent PR